src/os/bluestore/BlueStore.cc

```diff
@@ -4985,6 +4985,8 @@ int BlueStore::_setup_block_symlink_or_file(
 
     if (!epath.compare(0, strlen(SPDK_PREFIX), SPDK_PREFIX)) {
       int fd = ::openat(path_fd, epath.c_str(), flags, 0644);
+      char buf[40];
+      string::size_type pos;
```

Comment:
might want to move these definitions to where they are used for the first time.

Reply:
updated